x/ccv/provider/keeper/hooks.go

```diff
@@ -35,8 +36,63 @@ func (k *Keeper) Hooks() Hooks {
 func (h Hooks) AfterUnbondingInitiated(ctx sdk.Context, id uint64) error {
 	var consumerChainIDS []string
 
+	// get validator address from unbonding operation
+	unbondingType, found := h.k.stakingKeeper.GetUnbondingType(ctx, id)
+	vadAddrBech32 := ""
+	if !found {
+		ctx.Logger().Error("undefined type for unbonding operation id: %d", id)
```

Comment:
Should this and the ones below return an actual error?

Reply:
Returning an error here can potentially halt the provider when the hook is called after a validator starts unbonding in [val_state_change.go](https://github.com/cosmos/cosmos-sdk/blob/173f2ea773b47aaa99f878a58fdb6e09f0fbd717/x/staking/keeper/val_state_change.go#L354). So my suggestion is to stick to just logging those errors from the hook's perspective without returning any.

Reply:
That makes sense. Thanks!